### DISCUSSION TITLE
auto select in Multi Select

### DISCUSSION THREAD

**Discussion by yassermohammad** [wireui/wireui#314] [Q&A] [answered]
I am using Multi Select:

![Screenshot from 2022-06-13 14-55-27](https://user-images.githubusercontent.com/34732441/173334315-ef8539b8-141b-4ee7-a121-db7f05caf304.png)

 I want to know how can set an Item (for example Daily) as selected item automatically at the first of loading the page by default?

**Comment by ph7jack** [comment, marked as answer]
@yassermohammad Set the **'Daily'** value to **maintenance_frequency** property on the Livewire class

```
<?php

....

MyComponent extends Livewire\Component {
    public $maintenance_frequency = 'Daily';
}
```

**Reply by yassermohammad** [reply]
It is my class:

![Screenshot from 2022-06-13 18-38-17](https://user-images.githubusercontent.com/34732441/173372714-4b309ce5-757d-465a-b8a2-6573186044c1.png)


and it is my blade:
![Screenshot from 2022-06-13 18-39-31](https://user-images.githubusercontent.com/34732441/173372774-e260532b-8ba1-4edb-a641-8583b9543383.png)

but still the field is empty:

![Screenshot from 2022-06-13 18-40-44](https://user-images.githubusercontent.com/34732441/173373056-b63a3d3c-89b6-4cac-b547-6e3b2230836e.png)

**Reply by ph7jack** [reply]
You're using the multiselect, so your model must be an array
```
MyComponent extends Livewire\Component {
    public $maintenance_frequency = ['Daily'];
}
```

**Comment by yassermohammad** [comment]
Thank you
